pinot-core/src/main/java/org/apache/pinot/core/operator/docvalsets/ProjectionBlockValSet.java

```diff
@@ -127,6 +128,9 @@ public float[] getFloatValuesSV() {
     try (InvocationScope scope = Tracing.getTracer().createScope(ProjectionBlockValSet.class)) {
       recordReadValues(scope, DataType.FLOAT, true);
       return _dataBlockCache.getFloatValuesForSVColumn(_column);
+    } catch (RuntimeException re) {
+      //add column information, then later enrich it with table name
```

Comment:
> then later enrich it with table name

I presume this means we don't have access to the table name information in this class currently?

Reply:
I think table name was missing when I debugged some cases.
Let me double check .